index.html

```diff
@@ -865,10 +919,86 @@ <h4>issuance</h4>
 <section id="conformance"></section>
 
 <section>
-<h2>Privacy Considerations</h2>
+  <h2>Privacy Considerations</h2>
+  <p>
+    Verifiable Credentials often contain sensitive information that should be protected 
+    to ensure the privacy and security of individuals. This section outlines some privacy 
+    considerations that implementers and users of this specification should be aware of.
+  </p>
+  <p>
+    Implementers SHOULD note and abide by all privacy considerations called out in the [[VC-DATA-MODEL]].
```

Comment:
I agree with the intention here, but not sure about normative reference to an informative section... 
<img width="357" alt="Screen Shot 2023-04-28 at 4 39 18 PM" src="https://user-images.githubusercontent.com/8295856/235258791-6ce2a71c-a820-49ec-b011-71e31bf5f17a.png">

Reply:
yeah, we may need to make this normative if possible... for now I would say let's keep the desired language and adjust if needed - opened #82 to track this